### PR TITLE
Fixes #10278: Document usage of file sharing between nodes

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/10278

### CODE REVIEW

**Comment by peckpeck** [comment]
shouldn't we put a link to this page in the generic method documentation ?

File: 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+==== Share files between nodes
+
+Rudder 4.1 introduced a way to share files from one node to another.
+It allows a node to send a file to its relay, which will make it available
+for another target node, that has to to specifically download it.
+
+This file sharing method is secured by:
+
+* Control of uploaded file signature from the server, to check it matches node's private key
```
Comment (peckpeck) [inline 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt]: *by* the server

File: 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Same mechanism as standard file copy in Rudder to download the shared file from the server
```
Comment (peckpeck) [inline 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt]: Missing a '.' or a link work.

File: 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+It also includes a ttl mechnanism that allows sharing a file for a limited amount of time.
+
+To use this feature, two generic methods are available in the technique editor:
+
+* <<sharedfile_from_node, sharedfile_from_node>>: To download a file shared from another node.
+* <<sharedfile_to_node, sharedfile_to_node>>: To make a file available to another node.
+
+See the documentation of these methods for details about the required parameters.
```
Comment (peckpeck) [inline 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt]: I think you need an example of 2 matching calls here.

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
+==== Share files between nodes
+
+Rudder 4.1 introduced a way to share files from one node to another.
+It allows a node to send a file to its relay, which will make it available
+for another target node, that has to to specifically download it.
+
+This file sharing method is secured by:
+
+* The control of uploaded file signature by the server, to check it matches node's private key.
```
Comment (peckpeck) [inline 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt]: the source node's private key

File: 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+It also includes a ttl mechanism that allows sharing a file for a limited amount of time.
+
+To use this feature, two generic methods are available in the technique editor:
+
+* <<sharedfile_from_node, sharedfile_from_node>>: To download a file shared from another node.
+* <<sharedfile_to_node, sharedfile_to_node>>: To make a file available to another node.
+
+See the documentation of these methods for details about the required parameters, 
+ans especially <<sharedfile_to_node, sharedfile_to_node>> for a complete usage example.
```
Comment (peckpeck) [inline 23_manage_your_it/20_how_to/20_share_files_between_nodes.txt]: *and*

**Comment by amousset** [comment]
Commit modified

**Pull request merged**